packages/sdk-client-v3/tests/concurrent-modification.test/concurrent-modification-middleware.test.ts

```diff
@@ -39,7 +39,7 @@ describe('Concurrent Modification Middleware.', () => {
     const request = createTestRequest({ body: { version: 4 } })
     const response = createTestResponse({
       statusCode: 409,
-      error: { body: { errors: [{ currentVersion: 5 }] } },
```

Comment:
Shouldn't this be an error response?

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/617/commits/82ef8ac299dea3ed4e3f7c7e5184e34213524472#diff-51569edfbd3a5b4db84c8b79a0bd1cb94ebc092a0ebdd27387d15b683f310215R40